internal/config/v1/config_data.go

```diff
@@ -81,13 +81,10 @@ func (c *ConfigData) IsFalcoEbpfEngine() bool {
 func (c *ConfigData) setFalcoSyscallFilter() {
 	if c.IsFalcoEbpfEngine() {
 		for i := range c.ServicesList {
-			switch v := c.ServicesList[i].Data.(type) {
-			case SnifferData:
-				for j := range v.FeatureList {
-					if v.FeatureList[j].Name == SNIFFER_SERVICE_RELEVANT_CVES {
-						falcoSyscallFilter = append(falcoSyscallFilter, []string{"open", "openat", "execve", "execveat"}...)
-					}
-				}
+			features := c.ServicesList[i].Data.(map[string]interface{})["features"]
```

Comment:
all these casts are really ugly, and they don't provide compile-time safety
can we do better?